building-with-ocean/asset-hosting.md

```diff
@@ -1,51 +1,142 @@
 ---
 title: Publish assets using hosting services
```

Comment:
If I am not wrong, this page is displayed under the using-ocean-market section. Let's move it in that folder please